src/index.html

```diff
@@ -7,12 +7,77 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
```

Comment:
you don't need `img` tag, use `background-image: url()` according to requirements

Reply:
but in  qna chat was written that we need to use this requirements  with the stars and card__image we can add in html🙄

Reply:
sorry, yes, everything is correct 👍